designer/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/validation/PrettyValidationErrors.scala

```diff
@@ -144,6 +144,24 @@ object PrettyValidationErrors {
           s"There is more than one output with '$name' name defined in the fragment, currently this is not allowed",
           "Please check fragment definition"
         )
+      case InitialValueNotPresentInPossibleValues(paramName, _) =>
+        node(
+          s"The initial value provided for parameter '$paramName' is not present in the parameter's possible values list",
+          "Please check fragment definition",
```

Comment:
This exception is now used in fragment validation. Shouldn't we check it also for other components? Is it ok to suggest in description to check fragments only?

Reply:
The only other way to define a parameter that MUST be an element of a fixed list is through file config/UIConfigProvider, and there's currently no validation of what's provided there

we could in the future implement some validation of that, but it's definitely not for this PR